### PR TITLE
feature/send-email

### PR DESCRIPTION
**Pull request by GregVaz**
### Send email notifications for every book you delete.

image
image

### CODE REVIEW

File: mail/mail.mjs [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+</div>
+</div>
+</div>
+</td>
+</tr>
+</tbody>
+</table>
+</body>
+</html>`;
+};
```
Comment (jvargas98) [inline mail/mail.mjs]: Missing empty line at the end of the file

File: mail/mailer.mjs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import nodemailer from 'nodemailer';
+import { google } from 'googleapis';
+import template from './mail.mjs';
+import dotenv from 'dotenv';
+dotenv.config();
```
Comment (jvargas98) [inline mail/mailer.mjs:6]: Missing empty line after imports/requres
```suggestion

dotenv.config();
```

File: mail/mailer.mjs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+let transport = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    type: 'OAuth2',
+    user: process.env.EMAIL_USER,
+    clientId: process.env.EMAIL_CLIENT_ID,
+    clientSecret: process.env.EMAIL_CLIENT_SECRET,
+    refreshToken: process.env.EMAIL_REFRESH_TOKEN,
+    accessToken: accessToken
```
Comment (jvargas98) [inline mail/mailer.mjs]: Just with the accessToken works
```suggestion
    accessToken,
```

File: mail/mailer.mjs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    subject: create ? 'A new book was registered' : 'A book was deleted', // Subject line
+    generateTextFromHTML: true,
+    html: html_body
+  };
+
+  transport.sendMail(message, function(err, info) {
+    if (err) console.log(err);
+    else console.log(info);
+  });
+}
```
Comment (jvargas98) [inline mail/mailer.mjs]: please add a line break at the end of the file

**Review by jvargas98 — COMMENTED** [review]
Good job @GregVaz!

File: mail/mail.mjs [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+return `<!DOCTYPE html>
+
+<html>
+<head>
+<meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+<meta content="width=device-width" name="viewport"/>
+<meta content="IE=edge" http-equiv="X-UA-Compatible"/>
+<link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet" type="text/css"/>
+<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css"/>
+<style type="text/css">
```
Comment (LoginALee) [inline mail/mail.mjs:11]: It would be cool to have the css file apart

**Review by LoginALee — APPROVED** [review]
Good job Goyo :rocket:

File: mail/mail.mjs (hunk 1 above)
Comment (GregVaz) [reply mail/mail.mjs]: Ok, thanks :+1:

File: mail/mailer.mjs (hunk 3 above)
Comment (GregVaz) [reply mail/mailer.mjs]: Thanks Jorge :rocket:

File: mail/mail.mjs (hunk 5 above)
Comment (GregVaz) [reply mail/mail.mjs:11]: Yeah it would be really cool haha, but I do not know what the path to link the css file would be, anyway I will check it

**Pull request merged**